integrations/aws-iam/nango.yaml

```diff
@@ -0,0 +1,45 @@
+integrations:
+    aws-iam:
+        actions:
+            create-user:
+                description: Creates a user in AWS IAM.
+                output: User
+                endpoint: POST /users
+                input: AWSCreateUser
+            delete-user:
+                description: |
+                    Delete an existing user in AWS IAM. When you delete a user, you must delete the items attached to the user manually, or the deletion fails.
+                endpoint: DELETE /users
+                output: SuccessResponse
+                input: UserNamentity
+        syncs:
+            users:
+                runs: every day
+                description: |
+                    Fetches a list of users from AWS IAM
+                output: User
+                sync_type: full
+                track_deletes: true
+                endpoint: GET /users
+
+models:
+    # Generic
+    UserNamentity:
+        userName: string
+    SuccessResponse:
+        success: boolean
+    ActionResponseError:
+        message: string
+
+    User:
+        id: string
+        firstName: string
+        lastName: string
+        email: string
+    CreateUser:
+        firstName: string
+        lastName: string
+        email: string
+    AWSCreateUser:
```

Comment:
AWS-IAM handles users operation differently, in order for us to perform these actions, we should use `userName`  when creating and deleting users. Should we add this to the `users` sync?